OMCompiler/Compiler/NFFrontEnd/NFLookup.mo

```diff
@@ -1016,5 +1035,28 @@ algorithm
   end match;
 end makeInnerNode;
 
+function loadLibrary
+  input String name;
+  input InstNode scope;
+  output InstNode libraryNode;
+protected
+  String modelica_path;
+  Absyn.Program aprog;
+  Absyn.Class acls;
+  SCode.Element scls;
+  Class cls;
+algorithm
+  modelica_path := Settings.getModelicaPath(Testsuite.isRunning());
+  aprog := Absyn.Program.PROGRAM({}, Absyn.Within.TOP());
+  (aprog, true) := BackendInterface.loadModel({(Absyn.Path.IDENT(name), "", {"default"}, false)},
+    modelica_path, aprog, true, true, true, false);
+  acls := List.find(aprog.classes, function AbsynUtil.isClassNamed(inName = name));
+  scls := AbsynToSCode.translateClass(acls);
+  libraryNode := InstNode.new(scls, scope);
+  cls := InstNode.getClass(scope);
+  cls := Class.classTreeApply(cls, function ClassTree.appendClasses(clsNodes = {libraryNode}));
+  InstNode.updateClass(cls, scope);
+end loadLibrary;
```

Comment:
I suppose it would be good if it remained loaded also after the command (like we do for uses-annotations)

Reply:
I suppose I should use SymbolTable.setAbsyn like Main does when loading libraries, though it's a bit of a pain since that's also part of the backend.

Reply:
I just realised that libraries loaded due to uses-annotations in the loaded library isn't added to the top scope though, so it will still try to load Modelica even though it's already loaded. I need to check which classes haven't been added yet and add all of them to the top scope.

Reply:
So we need to check for loaded libraries in OMEdit after `translateModel` and `instantiateModel`?

Reply:
Guess what? We are already doing this in OMEdit so no changes required :)